data/tokens.json

```diff
@@ -472,6 +472,39 @@
       }
     }
   },
-  "Light": {},
+  "color_tokens": {
+    "text": {
+      "base": {
+        "value": "{colors.grey.800}",
+        "type": "color"
+      },
+      "emphasis": {
+        "value": "{colors.grey.800}",
```

Comment:
Is this and `subdued` meant to have the same value as `base`?

Reply:
Updated now :)